integrations/woopra/lib/index.js

```diff
@@ -0,0 +1,133 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+
+var each = require('component-each');
+var integration = require('@segment/analytics.js-integration');
+var toSnakeCase = require('to-snake-case');
+var is = require('is');
+var foldl = require('@ndhoule/foldl');
+var json = require('json3');
+var isostring = require('isostring');
+var time = require('unix-time');
+
+/**
+ * Expose `Woopra` integration.
+ */
+
+var Woopra = module.exports = integration('Woopra')
+  .global('woopra')
+  .option('domain', '')
+  .option('cookieName', 'wooTracker')
+  .option('cookieDomain', null)
+  .option('cookiePath', '/')
+  .option('ping', true)
+  .option('pingInterval', 12000)
+  .option('idleTimeout', 300000)
+  .option('downloadTracking', true)
+  .option('outgoingTracking', true)
+  .option('outgoingIgnoreSubdomain', true)
+  .option('downloadPause', 200)
+  .option('outgoingPause', 400)
+  .option('ignoreQueryUrl', true)
+  .option('hideCampaign', false)
+  .tag('<script src="//static.woopra.com/js/w.js">');
+
+/**
+ * Initialize.
+ *
+ * http://www.woopra.com/docs/setup/javascript-tracking/
+ */
+
+Woopra.prototype.initialize = function() {
+  /* eslint-disable */
```

Comment:
I guess this is supposed to be normal but if we could have a comment about the where it come from and why.